torch/utils/_pytree.py

```diff
@@ -131,7 +133,8 @@ def _register_pytree_node(
     to_dumpable_context: Optional[ToDumpableContextFn] = None,
     from_dumpable_context: Optional[FromDumpableContextFn] = None,
 ) -> None:
-    """
+    """Register a container-like type as pytree node.
+
```

Comment:
Let's delete to_str_fn and maybe_from_str_fn for the torch.utils._pytree.register_pytree_node API, since those are deprecated and we are creating a new api here.